tests/store/test_file_store.py

```diff
@@ -25,16 +25,15 @@ class TestFileStore(unittest.TestCase):
 
     def setUp(self):
         self._create_root(TestFileStore.ROOT_LOCATION)
-        self.maxDiff = None
 
     def _create_root(self, root):
         self.test_root = os.path.join(root, "test_file_store_%d" % random_int())
         os.mkdir(self.test_root)
-        self.experiments = [random_int(100, int(1e9)) for _ in range(3)]
-        self.exp_data = {}
-        self.run_data = {}
+        self.experiments = [str(random_int(100, int(1e9))) for _ in range(3)]  # noqa
```

Comment:
What caused the need for the `#noqa`

Reply:
The linter checks for properties set outside of the __init__, which is convention in pytest since you create them in def setup(self):, which is why the #  noqa was needed

With pytest, assert has the same rich printouts that assertEqual and co. have without introducing test specific functions. 

I undid the whole move to pytest because assertCountEqual is not supported by pytest and the need for #noqa.